teuthology/orchestra/run.py

```diff
@@ -102,6 +102,15 @@ def wait(self):
 
         status = self._get_exitstatus()
         self.exitstatus = self.returncode = status
+        for stream in ('stdout', 'stderr'):
+            if hasattr(self, stream):
+                stream_obj = getattr(self, stream)
+                # Despite ChannelFile having a seek() method, it raises
+                # "IOError: File does not support seeking."
+                if hasattr(stream_obj, 'seek') and \
+                        not isinstance(stream_obj, ChannelFile):
+                    stream_obj.seek(0)
```

Comment:
After the process completes, ``stream_obj.read()`` will always return ``''`` since it's still pointed at the end. The ``seek(0)`` points it back at the beginning, which is what callers will expect.